baselines/driving_smarts/benchmark.py

```diff
@@ -0,0 +1,140 @@
+# smarts/benchmark/driving_smarts_competition_benchmark.py
```

Comment:
Many files named `benchmark`, thus perhaps simply name this file `SMARTS/baselines/driving_smarts/run.py` .

Reply:
I moved it to `smarts/benchmark/entrypoints/benchmark_runner_v0.py` so that `scl benchmark` can use it.